test/fishjam/cluster/api_test.exs

```diff
@@ -139,60 +124,54 @@ defmodule Fishjam.Cluster.ApiTest do
     assert {:ok, %HTTPoison.Response{status_code: 201, body: body}} =
              HTTPoison.post("http://#{fishjam_instance}/room", request_body, @post_headers)
 
-    body
+    room_data = body |> Jason.decode!() |> Map.fetch!("data")
+    room_node = get_fishjam_address(room_data)
+
+    {room_data, room_node}
```

Comment:
Could, instead of a tuple, return a map with fields like `data,` `node,` `id` etc, and then pattern match in a specific line of code.